src/CBot/CBotUtils.cpp

```diff
@@ -245,6 +245,12 @@ bool CharInList(const char c, const char* list)
 
 std::string CodePointToUTF8(unsigned int val)
 {
+    // nota : similare with StrUtils::UnicodeCharToUtf8 (with traces but CBotErrUnicodeName)
```

Comment:
"note" -> note
"similare" -> similar
I don't really understand what this comment wants to tell.